### PR TITLE
TECH-805 remove old config for organizing hub

### PR DESCRIPTION
**Pull request by ericbroder**

### CODE REVIEW

File: local_groups/urls.py [hunk 1]
```diff
@@ -16,74 +16,74 @@
...
-                        name='password_reset_done',
-                    ),
-                ])),
-            ])),
-            url(r'^verify-email/', include([
                 url(
-                    r"^confirm/(?P<key>[-:\w]+)/$",
-                    VerifyEmailConfirmView.as_view(),
-                    name="groups-verify-email-confirm"
+                    r'^confirm/(?P<uidb64>[0-9A-Za-z_\-]+)/(?P<token>[0-9A-Za-z]{1,13}-[0-9A-Za-z]{1,20})/$',
```
Comment (cjmabry) [inline local_groups/urls.py]: What's happening here? Wasn't this just a config change?

Comment (ericbroder) [reply local_groups/urls.py]: This should all just be an indentation change. Github is no bueno at displaying trivial indent changes 😞

File: local_groups/urls.py [hunk 2]
```diff
@@ -16,74 +16,74 @@
     url(r'^join-us-on-slack', SlackInviteView.as_view())
 ]
 
-if settings.BSD_LOGIN_ENABLED:
```
Comment (ericbroder) [inline local_groups/urls.py]: this is the main change, removing this line, then un-indenting the stuff below

File: local_groups/urls.py (hunk 1 above)
Comment (cjmabry) [reply local_groups/urls.py]: Oh Okay I see now, thanks for clarifying!

**Pull request merged**